File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/openTripMap/Sources.java
package com.tammamkhalaf.myuaeguide.categories.openTripMap;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Sources {

    @SerializedName("geometry")
    @Expose
    private String geometry;
    @SerializedName("attributes")
    @Expose
    private List<String> attributes = null;

    public String getGeometry() {
        return geometry;
    }

    public void setGeometry(String geometry) {
        this.geometry = geometry;
    }

    public List<String> getAttributes() {
        return attributes;
    }

    public void setAttributes(List<String> attributes) {
        this.attributes = attributes;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/openTripMap/Sources.kt
package com.tammamkhalaf.myuaeguide.categories.openTripMap

import com.google.gson.annotations.Expose

class Sources {
    @SerializedName("geometry")
    @Expose
    private var geometry: String? = null

    @SerializedName("attributes")
    @Expose
    private var attributes: List<String>? = null
    fun getGeometry(): String? {
        return geometry
    }

    fun setGeometry(geometry: String?) {
        this.geometry = geometry
    }

    fun getAttributes(): List<String>? {
        return attributes
    }

    fun setAttributes(attributes: List<String>?) {
        this.attributes = attributes
    }
}